### ISSUE TITLE
Connection Problems ( Instabillity )

### ISSUE THREAD

**Issue by gbauso** [Azure/azure-signalr#281]
Hi guys,

I have been using the service with .NET Framework 4.6.1 ( with Version 1.0.0-preview1-10264 ) and JS library 2.4.0 and sometines I had problems with the connection ( "No server avaliable" ) and I had to restart the server and it is back working. 

![image](https://user-images.githubusercontent.com/15365659/49236467-11bd7980-f3f4-11e8-9e45-2e92e7e6dcf0.png)
![image](https://user-images.githubusercontent.com/15365659/49236525-331e6580-f3f4-11e8-80b1-17f0066053e8.png)

**Comment by gbauso** [comment]
More Informations:

- Our app is hosted on Azure.
- When I start my local application, the service strangely back to work.

![image](https://user-images.githubusercontent.com/15365659/49237512-429eae00-f3f6-11e8-8cad-dc01650b9e53.png)

**Comment by gbauso** [comment]
Hi Vicancy,

I followed your advice, and changed my Startup and it backs to work, but when we put in production we continue to handle with instability.

This is the line that I've added on Startup.

`app.MapAzureSignalR(this.GetType().FullName);`

And in my connection strings section I added the line as it described on this repo tutorial and the value was copied from azure service.

`<add name="Azure:SignalR:ConnectionString" connectionString="Endpoint=...;AccessKey=..."/>`

This is the azure's service dashboard

![image](https://user-images.githubusercontent.com/15365659/50010060-5dede980-ffb0-11e8-862d-49772c3affcb.png)

And it's a error that appears on navigator

![image](https://user-images.githubusercontent.com/15365659/50010095-73fbaa00-ffb0-11e8-84b5-31d694b66707.png)
